### PR TITLE
DBaaS Guide - Backup and Restore with CLI tools

### PR DESCRIPTION
**Pull request by herr-hxp**
A DBaaS Guide on how to backup and restore PostgreSQL, MariaDB and MySQL databases using the official cli tools.
Also restructured the "guides" dir and files a bit to have it work as a collection of pages rather than a single page so that we can add more guides in the future.

### CODE REVIEW

**Comment by mijanss** [comment]
A couple of suggestions, could be good to mention that it seems to be that postgres backup tool pg_dump does need to be at least on the same version as on the server.

Maybe add flags for user and password to the mariadb and mysql backup/restore tools.
For example: `-u (dbuser) -p`

**Comment by herr-hxp** [comment]
> A couple of suggestions, could be good to mention that it seems to be that postgres backup tool pg_dump does need to be at least on the same version as on the server.
> 
> Maybe add flags for user and password to the mariadb and mysql backup/restore tools. For example: `-u (dbuser) -p`

Specifically omitted user and password with the warning that it's not recommended. Links and information about using options file are included. I will add a note about pg versions for clarity.

File: content/en/docs/dbaas/guides/backup-and-restore-cli.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+--skip-lock-tables    Disable the default setting to lock tables
+```
+
+For a full list of options, check out the documentation under [MariaDB Documentation](#mariadb-documentation).
+
+```shell
+mariadb-dump -h mydatabaseserver -B mydatabase --quick --single-transaction --skip-lock-tables > dump.sql
+```
+
+> It is not recommended to specify the password via the command line. Consider using an option file instead, by default the client checks the user's home directory for a file named `.my.cnf`. You can read more about Option Files in the official documentation linked under [MariaDB Documentation](#mariadb-documentation).
```
Comment (Tipset) [inline content/en/docs/dbaas/guides/backup-and-restore-cli.md:89]: maybe change 'Option Files' to 'option files'.

**Review by Tipset — COMMENTED** [review]
check my comments

File: content/en/docs/dbaas/guides/backup-and-restore-cli.md (hunk 1 above)
Comment (herr-hxp) [reply content/en/docs/dbaas/guides/backup-and-restore-cli.md]: done

**Comment by mijanss** [comment]
LGTM

**Pull request merged**